src/main/java/net/fabricmc/loom/task/AbstractLoomTask.java

```diff
@@ -26,8 +26,14 @@
 
 import org.gradle.api.DefaultTask;
 
+import net.fabricmc.loom.LoomGradleExtension;
+
 public abstract class AbstractLoomTask extends DefaultTask {
 	public AbstractLoomTask() {
 		setGroup("fabric");
 	}
+
+	LoomGradleExtension getExtension() {
```

Comment:
Good idea, not sure why this wasnt done before.

Reply:
Make this protected for addons to loom?